src/backend/distributed/executor/citus_custom_scan.c

```diff
@@ -682,11 +682,13 @@ RegenerateTaskForFasthPathQuery(Job *workerJob)
 	}
 
 	bool isLocalTableModification = false;
+	bool isLocalShardExecution = false;
```

Comment:
maybe we can name this variable as "delayedFastPath" as we named it in the function's signature